app/views/fields/bulletpoint-list.test.js

```diff
@@ -60,7 +61,8 @@ describe('bulletpoint-list', () => {
 
         const additionalAdvice = $('[data-test-id="bulletpoint-list-additional-advice"]');
 
-        expect(additionalAdvice.find('.nhsuk-label').text().trim()).toEqual('some additional advice for question');
+        expect(additionalAdvice.find('.nhsuk-label')
```

Comment:
I think like the main advice above. Perhaps not necessary to do the additional find on `.nhsuk-label`

Reply:
That will only find the div. Not the label nunjucks component. We might be best just looking for the actual html element rather than that particular class?